.github/workflows/build.yml

```diff
@@ -28,6 +33,65 @@ jobs:
         run: make pip
       - name: Build and Test
         run: make build-and-test
+  run-integration-tests:
+    runs-on: ubuntu-20.04
+    # container: ubuntu:22.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install required os level applications
+        run: |
+          sudo apt update -y
+          sudo apt install -y curl git nodejs python-is-python3 python3-pip
```

Comment:
As part of bringing these changes to a mergeable state, I think we ought to disambiguate steps needed for act.

* In order to get act to pass, I sourced various dependencies that I believe are _not_ needed for github.
* Disambiguation here means creating 1 or more steps dedicated to act dependencies.  Goal is to make it easier to maintain the build.
* I _think_ the following dependencies are act-specific though it requires a round of testing:
  * `curl`, `nodejs`, `python3-pip`
  * Docker install (e.g. `docker-ce`)
  * _Setup docker compose_